nautobot/core/views/generic.py

```diff
@@ -80,7 +80,9 @@ def get_extra_context(self, request, instance):
         Returns:
             dict
         """
-        return {}
+        return {
+            "active_tab": request.GET.get("tab", "main"),
+        }
```

Comment:
Do we have existing object-detail views that override `get_extra_context` but don't call `super()` that need to be fixed to account for this now?

Reply:
As far as I'm aware there is housekeeping that can be done to simplify what other views are doing but since they were manually setting `active_tab` in a lot of places already this shouldn't break any existing views.